10.9-libcxx/stable/main/finkinfo/sci/openblas.info

```diff
@@ -1,7 +1,7 @@
 # -*- coding: ascii; tab-width: 4 -*-
 Info2: <<
 Package: openblas
-Version: 0.3.19
+Version: 0.3.20
```

Comment:
0.3.21 supposedly has a fix for linking on M1 CPUs.

Reply:
Forgot to push that...